src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -368,31 +369,31 @@ private int ParseAuditLevel()
 
             if (auditLevel == null)
             {
-                return 1;
+                return 0;
```

Comment:
>return 0; [](http://example.com/codeflow?start=16&length=9)

Could we add a comment on when auditLevel is expected to be null? I'm guessing this is the default case, so it would be good to comment that and also comment why the default case was decided to be equivalent to low.